### ISSUE TITLE
Warnings compiling solidity contracts

### ISSUE THREAD

**Issue by mfornet** [near/rainbow-bridge#491]
Compilation of solidity contracts emits several warnings. To see this locally use:

```
cd contracts/eth/nearbridge
./dist.sh
```

### Warnings

- [x] [truffle-flattener](https://github.com/nomiclabs/truffle-flattener) is using old version of solidity-parser. This warning is already solved: https://github.com/solidity-parser/parser/issues/19

```
 Warning: Accessing non-existent property 'INVALID_ALT_NUMBER' of module exports inside circular dependency
```

- [ ] [No License](https://forum.openzeppelin.com/t/solidity-0-6-8-introduces-spdx-license-identifiers/2859) on solidity code triggers warning. Consider using [this approach](https://github.com/nomiclabs/truffle-flattener/issues/55#issuecomment-679157423) (Only keep one License rather than removing all of them).
```
dist/Borsh.full.sol: Warning: SPDX license identifier not provided in source file. Before publishing, consider adding a comment containing "SPDX-License-Identifier: <SPDX-License>" to each source file. Use "SPDX-License-Identifier: UNLICENSED" for non-open-source code. Please see https://spdx.org for more information.
```

- [x] Some docstring [will stop working](https://blog.soliditylang.org/2020/07/28/solidity-v0.7.0-release-announcement/) on version 0.7.0 of solidity
```
dist/NearBridge.full.sol:529:17: Warning: Only state variables can have a docstring. This will be disallowed in 0.7.0.
        bytes32 epoch_id;           /// Epoch start hash of this block's epoch. Used for retrieving validator information
                ^------^
```

- [x] `Warning: Function state mutability can be restricted to pure`

```
dist/NearDecoder.full.sol:464:5: Warning: Function state mutability can be restricted to pure
    function decodeInitialValidators(Borsh.Data memory data) internal view returns(InitialValidators memory validators) {
    ^ (Relevant source part starts here and spans across multiple lines).
```

- [ ] Installing gyp/scrypt [view log](https://gist.github.com/mfornet/dec42dd22418396cd4c31dd74b61bf6e)

**Comment by KaiserKarel** [comment]
I can check this out this weekend.

**Comment by mfornet** [comment]
Some warnings solved on #553
